proseco/fid.py

```diff
@@ -128,6 +131,7 @@ def t_ccd(self):
     @t_ccd.setter
     def t_ccd(self, value):
         self.t_ccd_guide = value
+        self.t_ccd_acq = value
```

Comment:
In more detailed review and testing, _this_ was the change needed so that "get_fid_catalog" will mostly do the right thing if *just* a t_ccd is supplied.  In that case the temperature is assigned to both t_ccd_guide and t_ccd_acq and fid offsets will be applied based on t_ccd_acq by default.  This was already the default behavior for get_aca_catalog, but not for get_fid_catalog.

Reply:
Seems reasonable.